packages/components/src/Heading/Heading.tsx

```diff
@@ -17,8 +17,9 @@ type Props = {
   className?: TypographyProps<HeadingElement>["className"];
   color?: TypographyProps<HeadingElement>["color"];
   size: TypographyProps<HeadingElement>["size"];
-  weight?: TypographyProps<HeadingElement>["weight"];
   spacing?: TypographyProps<HeadingElement>["spacing"];
+  tabIndex?: number;
```

Comment:
As the comment in this file points out, we could add `React.HTMLAttributes<HTMLHeadingElement>` to the Props type to get all the attributes someone could pass in.

That makes sense to me, but I opted here for the smaller change.

Reply:
sg, I added a task for it: https://app.clubhouse.io/czi-edu/story/66248/use-react-htmlattributes-htmlheadingelement-in-heading-props